### ISSUE TITLE
Bear Settlement issue

### ISSUE THREAD

**Issue by savagesach** [jj248/RealmsInExile#809]
there's something specifically wrong with the bear building, where the blocker doesn't get removed which means the county can never get fully settled

![image](https://user-images.githubusercontent.com/86676697/228337750-ae901677-7735-4198-8f6b-d9b07bfe60e3.png)
![image](https://user-images.githubusercontent.com/86676697/228337771-c7dc3d0a-0a7c-47ad-bc09-54e7889ec8a1.png)
![image](https://user-images.githubusercontent.com/86676697/228337800-5bdd032e-e4dd-4094-bcf2-bf4297c0b63e.png)

**Comment by savagesach** [comment]
yeah it works for me now, I've no longer gotten the issue. this can be closed

**Comment by JukeJukestar** [comment]
We have gotten a few reports in public channels of this occurring. reopening.

![image](https://github.com/jj248/RealmsInExile/assets/76008882/f5ea4b10-89f1-4302-827a-1dbc922810b3)

I looked at the code myself and I cant see anything wrong with it - I dont believe its throwing any errors either.

**Comment by JukeJukestar** [comment]
@jj248 took the liberty to assign you, I've done some blocker stuff but this may be a base-game bug we have to find a workaround for
